test/remote/gateways/remote_quickbooks_test.rb

```diff
@@ -22,7 +22,7 @@ def setup
   def test_successful_purchase
     response = @gateway.purchase(@amount, @credit_card, @options)
     assert_success response
-    assert_equal 'Transaction Approved', response.message
+    assert_equal 'CAPTURED', response.message
```

Comment:
Ah - I see now. Do we want this to be the message, or do we still want "Transaction Approved"?